src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -83,7 +91,7 @@ export class HaSceneEditor extends SubscribeMixin(
 
   @property() public route!: Route;
 
-  @property() public sceneId?: string;
+  @property() public sceneId!: string;
```

Comment:
Why was this changed?

Reply:
This could be because I do not know enough about TypeScript.

I was getting the below warnings.
![image](https://user-images.githubusercontent.com/4094413/118261358-5269ba00-b4ab-11eb-8a00-1295c1381ba5.png)

Since duplication had already been implimented on the Scripts and Automations pages I looked at how it was achieved and replicated the top-right menu for Scenes.

### Automation Editor
https://github.com/home-assistant/frontend/blob/fe2fe7468f6e7b975e215646b6c4d5eec20dcd92/src/panels/config/automation/ha-automation-editor.ts#L78

https://github.com/home-assistant/frontend/blob/fe2fe7468f6e7b975e215646b6c4d5eec20dcd92/src/panels/config/automation/ha-automation-editor.ts#L176-L193

### Script Editor
https://github.com/home-assistant/frontend/blob/fe2fe7468f6e7b975e215646b6c4d5eec20dcd92/src/panels/config/script/ha-script-editor.ts#L64

https://github.com/home-assistant/frontend/blob/fe2fe7468f6e7b975e215646b6c4d5eec20dcd92/src/panels/config/script/ha-script-editor.ts#L159-L174


### Possible Alternative
Change back to optional
```suggestion
  @property() public sceneId?: string;
```

And use ! on the sceneId in the classMap:
https://github.com/home-assistant/frontend/pull/9175#discussion_r632457549
https://github.com/home-assistant/frontend/pull/9175#discussion_r632457625

Reply:
Cast `sceneId` to a boolean: `Boolean(this.sceneId)`

Reply:
Corrected for script and automation editor here: <https://github.com/home-assistant/frontend/pull/9184>

Reply:
Thanks, made the changes.